### PR TITLE
readme: Add a section on `overrideAttrs`

### PR DESCRIPTION
**Pull request by Patryk27**
Closes https://github.com/nix-community/naersk/issues/277.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,9 @@ naersk.buildPackage {
 Some of the options (described below) are used by Naersk to affect the building
 process, rest is passed-through into `mkDerivation`.
 
+Note that you shouldn't call `overrideAttrs` on a derivation built by Naersk
+(see the explanation below).
```
Comment (nmattia) [inline README.md:226]:
```suggestion
(see [Note on `overrideAttrs`](#note-on-overrideattrs) below).
```

**Review by nmattia — DISMISSED** [review]
Very nice! Thanks a lot

File: README.md (hunk 1 above)
Comment (Patryk27) [reply README.md]: true, true, added 🙂

**Review by nmattia — APPROVED** [review]
Looking really nice, thanks!

**Pull request merged**